src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.server
+
+import akka.actor.ActorSystem
+import cromwell.engine.WorkflowManagerActor
+
+trait WorkflowManagerSystem {
+  val systemName = "cromwell-system"
+  val actorSystem = ActorSystem(systemName)
+
+  actorSystem.registerOnTermination({
+    actorSystem.log.info(s"$systemName shutting down")
+  })
```

Comment:
Dunno, I copy/pasted it from something else :) You're probably right. Although look at you with the one liners! I'll poke at it tomorrow